### DISCUSSION TITLE
Code Snippet Highlighting in blog is working partially

### DISCUSSION THREAD

**Discussion by jaymehta20** [vercel/next.js#24382] [Help]
### What version of Next.js are you using?

10.1.3

### What version of Node.js are you using?

14.16.0

### What browser are you using?

Chrome

### What operating system are you using?

Windows 10

### How are you deploying your application?

Vercel

### Describe the Bug

I am trying to load a custom client-side script in the head section but it doesn't work the console shows 404.

![image](https://user-images.githubusercontent.com/23277339/115863118-a2c49d80-a452-11eb-9888-bdf6c698d3ff.png)

You can take a look at the code here:
https://github.com/jaymehta20/codegrain-blog-next

### Expected Behavior

The custom script should load which functions the navbar and animation. 
I am using Jquery cause I am converting my old HTML CSS JS site to NextJS

### To Reproduce

https://github.com/jaymehta20/codegrain-blog-next

**Comment by timneutkens** [comment]
Just checked and both files are served fine 🤔 It seems that you did not prefix the scripts though:

```
<script type="text/javascript" src="js/plugin.min.js"></script>
```

Should be:

```
<script type="text/javascript" src="/js/plugin.min.js"></script>
```

**Reply by jaymehta20** [reply]
Can you show me the output by toggling the nav

**Reply by jaymehta20** [reply]
![image](https://user-images.githubusercontent.com/23277339/115864878-28e1e380-a455-11eb-9101-5a4d3f84010e.png)
It still shows me this even when I tried this 
<script type="text/javascript" src="/js/plugin.min.js"></script>

**Reply by apollonian** [reply]
It's a typo (missing "s")! The file names in your [public directory](https://github.com/jaymehta20/codegrain-blog-next/tree/071f854/public/js) are:
1. function***s***.js
2. plugin***s***.min.js

**Reply by jaymehta20** [reply]
Thanks, it worked 😅

**Reply by jaymehta20** [reply]
I have a question, Is it ok if I put the script tags in _app.js, or does it cause any issues?

**Reply by jaymehta20** [reply]
Also, can you tell me how to add a CDN link in nextjs @apollonian

**Reply by jaymehta20** [reply]
I just noticed one more problem rn is the loader doesn't disappear instantly even when the website is loaded 
You can check it here:
https://blog.codegra.in/
@apollonian Thanks

**Reply by apollonian** [reply]
> I have a question, Is it ok if I put the script tags in _app.js, or does it cause any issues?

The layout component that you have would be a good place imo.

> add a CDN link 

Um, not sure if I understand this correctly. It would be similar to what you're doing currently?

**Reply by jaymehta20** [reply]
> Um, not sure if I understand this correctly. It would be similar to what you're doing currently?
i meant I want to load prismjs script from cdn to my code

**Reply by apollonian** [reply]
Do give https://github.com/FormidableLabs/prism-react-renderer a check if you're considering Prism for syntax highlighting!

**Reply by jaymehta20** [reply]
> Do give https://github.com/FormidableLabs/prism-react-renderer a check if you're considering Prism for syntax highlighting!

Hey, I can't seem to figure out how to implement that in our code can you help us out?
You can message us at https://www.instagram.com/codegra.in/

**Reply by jaymehta20** [reply]
@apollonian You can check the final result on https://blog.codegra.in/blog/4-less-known-html-tags.
It works for some code blocks but doesn't apply styles to others.

**Reply by apollonian** [reply]
Try replacing the code which isn't being highlighted with the one that is. Does it work after that? Could likely help pinpoint if something is wrong with the code/renderer.

**Reply by jaymehta20** [reply]
> Try replacing the code which isn't being highlighted with the one that is. Does it work after that? Could likely help pinpoint if something is wrong with the code/renderer.

Nope tried replacing it but the highlighting doesn't work

**Reply by jaymehta20** [reply]
This is the code that I used to change the p tag from contentful to pre tag
```
[BLOCKS.PARAGRAPH]: (node, children) => {
      console.log(node);
      if (
        node.content.length === 1 &&
        node.content[0].marks.find((x) => x.type === "code")
      ) {
        return <pre className="lang-html">{children}</pre>;
      }
      return <p>{children}</p>;
    },
```
